BasicSample/Assets/Shared/Scripts/RuntimeInfo.cs

```diff
@@ -59,6 +69,76 @@ private static string GetDisplayInfo()
             }
         }
 
+        private static string GetTrackingInfo()
+        {
+            var leftHandTracked = "Not tracked";
+            var rightHandTracked = "Not tracked";
+            var headTracked = "Not tracked";
+            var leftHandDevice = InputDevices.GetDeviceAtXRNode(XRNode.LeftHand);
+            var rightHandDevice = InputDevices.GetDeviceAtXRNode(XRNode.RightHand);
+            var headDevice = InputDevices.GetDeviceAtXRNode(XRNode.Head);
+            if (leftHandDevice.isValid && leftHandDevice.TryGetFeatureValue(CommonUsages.isTracked, out bool lTracked) && lTracked)
```

Comment:
can we make this to be 3 states?  "Not Found, Not Tracked, and Tracked"?

Reply:
Done